boilerplate/openshift/golang_osd_cluster_operator/standard.mk

```diff
@@ -28,6 +28,8 @@ OPERATOR_DOCKERFILE ?=build/Dockerfile
 BINFILE=build/_output/bin/$(OPERATOR_NAME)
 MAINPACKAGE=./cmd/manager
 
+CRDS_PATH ?= deploy/crds
```

Comment:
Maybe we want to discover instead. Something like

```
git ls-files | grep -v ^vendor/ | egrep '\.ya?ml'
```

(The python script is set up to accept files or directories.)